docs/UserGuide.md

```diff
@@ -387,12 +393,12 @@ Examples:
 
 Add multiple residents to an event based on the information given(name or room number), a resident is only expected to be given one piece of information.
 
-Format: `include INDEX r/INFORMATION [, MORE INFORMATION]`
+Format: `include INDEX [r/RESIDENTS]`
 
-* Resident information can be given in the form of name or room, but all has to be all rooms or all names
-* When adding multiple residents, each piece of information is separated by a comma
-* The information inputted is case-insensitive
-* If one or more of the given information is invalid, an error message is outputted and none of the residents are added to the event
+* Residents can be given in the form of name or room, but all has to be all rooms or all names
+* When adding multiple residents, each resident is separated by a comma
+* The resident inputted is case-insensitive
```

Comment:
Perhaps could be "names/rooms" instead of resident for clarity

Reply:
sounds good